Articles/Blog/2022-05-Software-Deployment-Bringing-resources-into-effective-acction.md

```diff
@@ -0,0 +1,194 @@
+centers can also benefit from each other by learning the Software Deployment 
+process, especially for centers that don’t have a well-established process. 
+
+We welcome the community to share their deployment strategy and/or best 
+practices for Software Stack Deployments. Let’s share game plans.
+
+
+### Author bio
+
+#### Shahzeb Siddiqui
+
+[Shahzeb Siddiqui](https://github.com/shahzebsiddiqui) is a HPC 
+Consultant/Software Integration Specialist at 
+[Lawrence Berkeley National Laboratory](https://www.lbl.gov/) at 
+[NERSC](http://nersc.gov/). He is part of 
+[User Engagement Team](https://www.nersc.gov/about/nersc-staff/user-engagement/) 
+that is responsible for engaging with NERSC user community through user support 
+tickets, user outreach, training, documentation. Shahzeb is part of the 
+[Exascale Computing Project (ECP)](https://www.exascaleproject.org/) in 
+[Software Deployment](https://www.exascaleproject.org/research-group/software-deployment-at-the-facilities/) (SD) group 
+where he is responsible for building Spack [Extreme-Scale Scientific Software Stack](https://e4s-project.github.io/) (E4S) at the DOE facilities. 
+
+Shahzeb Siddiqui started out his career in High Performance Computing (HPC) in 2012 
+at [King Abdullah University of Science and Technology](https://www.kaust.edu.sa/en) (KAUST) 
+while pursuing his Masters. His focus in HPC includes Parallel Programming, 
+Performance Tuning, Containers (Singularity, Docker), Linux system administration,
+Scientific Software Installation and testing, Scheduler Optimization, and 
+Job Metrics. Shahzeb has held multiple roles in his HPC career in the following 
+companies: Dassault-Systemes, Pfizer, Penn State, and IBM. Prior to 2012, he was 
+a software engineer holding multiple roles at Global Science & Technology, 
+Northrop Grumman, and Penn State.
+
+#### Sameer Shende
+
+Dr. Sameer Shende has helped develop the 
+[TAU Performance System](http://www.cs.uoregon.edu/research/tau/home.php), the 
+[Program Database Toolkit (PDT)](https://www.cs.uoregon.edu/research/pdt/home.php), 
+the [Extreme-scale Scientific Software Stack (E4S)](https://e4s.io) 
+and the HPCLinux distro. His research interests include tools and techniques for 
+performance instrumentation, measurement, analysis, runtime systems, 
+HPC container runtimes, and compiler optimizations. He serves as a 
+Research Associate Professor and the Director of the 
+[Performance Research Laboratory](https://nic.uoregon.edu/prl/home.php) at the 
+[University of Oregon](https://www.uoregon.edu/), 
+and as the President and Director of [ParaTools](https://www.paratools.com/),
+Inc., ParaTools, SAS, and ParaTools, Ltd. He leads the SDK project for the 
+Exascale Computing Project (ECP), in the Programming Models and Runtime (PMR). 
+He received his B.Tech. in Electrical Engineering from IIT Bombay, and his M.S.
+and Ph.D. in Computer and Information Science from the University of Oregon.
```

Comment:
@sameershende can you please confirm your bio and all the hyperlinks. One thing that confused me was all the different `Paratools` in your bio.

Reply:
okay i removed Paratools, Ltd. Please see recent commit